virttest/utils_misc.py

```diff
@@ -2062,6 +2062,8 @@ def get_qemu_version(params=None):
             version['update'] = int(search_result.group(3))
         if "rhev" in str(line).lower():
             version['is_rhev'] = True
+        if "mod" in str(line).lower():
```

Comment:
whole word 'module' is meaningful than  "mod" , change from "mod" to "module" ?

Reply:
no, still have some module named scrmod means scratch module